style.css

```diff
@@ -80,17 +80,722 @@ body {
   font-weight: 500;
 }
 
-ul {
+.about ul {
   display: flex;
   flex-wrap: wrap;
   margin: 10px 0 0 10px;
 }
 
-ul li {
+.about ul li {
   list-style: none;
   margin-left: 20px;
 }
 
-ul li a {
+.about ul li a {
   text-decoration: none;
 }
+
+/* Work Section */
+.work {
+  display: grid;
+  grid-template-columns: 1fr;
+}
+
+/* Card 1 */
+.card-1 {
+  width: 90%;
+  border: 1px solid var(--neutrals-n-40, #dfe1e6);
+  border-radius: 16px;
+  background: var(--neutrals-n-0, #fff);
+  margin: 0 auto;
+  margin-top: 114px;
+  padding: 15px 15px 30px 15px;
+
+  /* display: flex;
+  flex-direction: column;
+  /*align-items: center; */
+}
+
+.card-1 img {
+  width: 100%;
+}
+
+.tonic {
+  color: #172b4d;
+  font-size: 44px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 38px;
+  margin-top: 15px;
+  display: flex;
+  flex-direction: column;
+}
+
+.canop {
+  display: flex;
+  flex-flow: row wrap;
+  margin-top: 15px;
+}
+
+.can-info {
+  color: #344563;
+  font-size: 13px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 16px;
+}
+
+.point {
+  width: 8px;
+  height: 8px;
+  background: #c1c7d0;
+  border-radius: 5px;
+  margin: 3px 0 0 6px;
+}
+
+.back {
+  color: #7a869a;
+  font-size: 13px;
+  font-style: normal;
+  font-weight: 600;
+  line-height: 16px;
+  margin: 0 0 0 6px;
+}
+
+.card-1 .paranic {
+  display: flex;
+  width: 295px;
+  height: 76px;
+  flex-direction: column;
+  color: #344563;
+  font-size: 15px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 24px;
+  margin-top: 15px;
+}
+
+/* .languages {
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 10px;
+  gap: 8px;
+}
+
+.languages p {
+  color: #6070ff;
+  font-size: 13px;
+  font-style: normal;
+  font-weight: 500;
+  border-radius: 8px;
+  background-color: #ebebff;
+  padding: 5px 10px;
+} */
+
+.languages {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
+  margin-top: 15px;
+}
+
+.card-1 .languages li {
+  list-style: none;
+}
+
+.card-1 .languages li a {
+  text-decoration: none;
+  display: flex;
+  padding: 4px 12px;
+  align-items: center;
+  border-radius: 8px;
+  background: var(--primary-b-50, #ebebff);
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 16px;
+  letter-spacing: 0.36px;
+}
+
+.see {
+  border-radius: 8px;
+  border: 1px solid #6070ff;
+  margin-top: 25px;
+  padding: 20px;
+  background-color: white;
+  color: #396df2;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 14px;
+  letter-spacing: 0.51px;
+}
+
+.see:hover {
+  background-color: #396df2;
+  color: white;
+}
```

Comment:
Good job implementing the `hover` state button interaction 👍 

As requested by the previous reviewer, please implement the button styling:  `Pressed` also (see design) 🤗🛠

![image](https://github.com/mar4444/re-portfolio/assets/78251927/8680df54-f98e-4bba-9842-1b3469680f05)


```CSS
.see:active {
 . . . 
}
```